### PR TITLE
Issue 7 implement global error handler

### PR DESCRIPTION
**Pull request by NirjoyDebnath**

### CODE REVIEW

File: src/utils/jwtHelper.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+};
+
+export const getToken = async (user: IUser): Promise<string> => {
+  const payload: IPayload = {
+    userName: user.UserName,
+    name: user.Name,
+    role: user.Role
+  };
+  const token =
+    'bearer ' +
```
Comment (edge555) [inline src/utils/jwtHelper.ts:21]: convention is 'Bearer' (B uppercase)

File: src/utils/jwtHelper.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    jwt.sign(payload, ENV.SecretKey || 'hello', {
+      expiresIn: ENV.JwtTokenExpire || '2d'
```
Comment (edge555) [inline src/utils/jwtHelper.ts:23]: signing should only use SecretKey

File: src/utils/jwtHelper.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+};
+
+export const verifyToken = async (
+  token: string | undefined
+): Promise<ITokenInfo> => {
+  if (!token) {
+    throw new appError(401, 'No token provided');
+  }
+  token = token.split(' ')[1];
+  const decodedToken = jwt.verify(token, ENV.SecretKey || 'hello');
```
Comment (edge555) [inline src/utils/jwtHelper.ts:35]: decoding should only use SecretKey

**Review by edge555 — APPROVED** [review]

File: src/utils/responses.ts [hunk 4]
```diff
@@ -5,22 +5,23 @@ import { jsonToPlainText, Options } from 'json-to-plain-text';
 const json2html = require('json2html');
 
 function contentNegotiation(type: string | undefined, data: object) {
```
Comment (Propo41) [inline src/utils/responses.ts:7]: method name: `negotiateContent` would be better

**Pull request merged**